lib/utils/stringHelper.js

```diff
@@ -1,11 +1,27 @@
+var _ = require('lodash');
+
 /**
  * Capitalize first letter in string
  * @param {String} string
  */
 function capitalizeFirstLetter(string) {
   return string.charAt(0).toUpperCase() + string.slice(1);
 }
-
+/**
+ *Capatalize every first letter in component name
+ *button-click --> Button-Click for Stroy book
+ * @param {*} string
+ * @returns
+ */
+function componentNameWithoutSpecialCharacter(string){
```

Comment:
This function can be much shorter and you should remove lodash. Please write it like this

```
/**
 * Removes special characters from string makes every camel case character uppercase
 * 
 * @param {String} str - String to transform
 * @returns {String}
 */
function componentNameWithoutSpecialCharacter(str){
  return str.split('-').map(capitalizeFirstLetter).join('')
}
```